content/en/05-reference/01-admin/04-deploy-options.md

```diff
@@ -3,9 +3,6 @@ title: "Deploy Options"
 metaTitle: "Reference | Admin | Deploy Options"
 metaDescription: "Configuration options for Pixie on deploy"
 order: 4
-redirect_from:
-    - /admin/deploy-options/
```

Comment:
can't we leave the first redirect intact? I don't see a reason to remove/break this